doc/generate-api-docs.sh

```diff
@@ -51,10 +51,29 @@ LIBS_SORTED=$(
 
 for LIB in $LIBS_SORTED; do
     cd $LIB
-    ARGS=""
-    if FEATURES="$(cargo read-manifest|jq -r '.metadata.docs.rs.features | join(",")' 2> /dev/null)"; then
-        ARGS="--features $FEATURES"
+    version=$(git tag --sort=taggerdate | grep $LIB'_' | tail -n1 | cut -d'_' -f2 | cut -d'v' -f2)
+    dependency=$LIB' = { version = "'$version'"'
+    features="$(cargo read-manifest|jq -c '.metadata.docs.rs.features')"
+    if [ $features != 'null' ]; then
+        dependency=$dependency', features = '$features
     fi
-    cargo doc --no-deps --lib $ARGS
+    dependency=$dependency' }'
     cd -
+
+    pushd $WORKSPACE
```

Comment:
Where is this set?

Reply:
It's an environment variable set for this process, should I check if its set before its used?

Reply:
Yes

Reply:
Done